ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/toolbar/DiskResourceViewToolbarImpl.java

```diff
@@ -320,14 +323,14 @@ public void onDiskResourceSelectionChanged(DiskResourceSelectionChangedEvent eve
         editCommentsMi.setEnabled(editCommentsMiEnabled);
         editInfoTypeMi.setEnabled(editInfoTypeMiEnabled);
 
-        copymetadataMi.setEnabled(metadataMiEnabled && isReadable(selectedDiskResources.get(0)));
-        savemetadatami.setEnabled(metadataMiEnabled && isReadable(selectedDiskResources.get(0)) );
+        copymetadataMi.setEnabled(metadataMiEnabled && isReadable(first_item));
+        savemetadatami.setEnabled(metadataMiEnabled && isReadable(first_item) );
         bulkmetadataMi.setEnabled(
-                metadataMiEnabled && isFolderSelect && (isOwner || isWritable(selectedDiskResources)));
+                metadataMiEnabled && isFolderSelect && (isOwner || isWriteable));
         selectmetadataMi.setEnabled(
-                metadataMiEnabled && isFolderSelect && (isOwner || isWritable(selectedDiskResources)));
+                metadataMiEnabled && isFolderSelect && (isOwner || isWriteable));
         doiMi.setEnabled(metadataMiEnabled && isFolderSelect && isOwner);
-        editmetadataMi.setEnabled(metadataMiEnabled && isReadable(selectedDiskResources.get(0)));
+        editmetadataMi.setEnabled(metadataMiEnabled && isReadable(first_item));
```

Comment:
Why not use the `isReadable` boolean already defined, instead of calling `isReadable(first_item)` each time?